### ISSUE TITLE
Missing import

### ISSUE THREAD

**Issue by jorgedch** [stsievert/salmon#140]
### Software paper
https://github.com/openjournals/joss-reviews/issues/4517

- [ ] **Functionality:**

1. The library `ipywidgets` is needed in `requirements.txt` to run the code in [Generating embeddings offline](https://docs.stsievert.com/salmon/offline.html#generate-embeddings)

**Comment by stsievert** [comment]
Do you have a more detailed traceback? I'm not seeing any mention of `ipywidgets` in the repo (both in #139 and on the master branch):

``` shell
(review-comments) $ pwd
/Users/scott/Developer/stsievert/salmon
(review-comments) $ grep -Hnri "ipywidgets" .
(review-comments) $ git checkout master
(master) $ grep -Hnri "ipywidgets" .
(master) $
```

**Comment by jorgedch** [comment]
Indeed, it's not going to show in the repo as it is a missing requirement in the packages indicated in `requirements.txt`. I got an error when running the example code in [Generating embeddings offline](https://docs.stsievert.com/salmon/offline.html#generate-embeddings) and it was solved after I added `ipywidgets` in `requirements.txt`. So the problem is fixed, it's only matter of adding the package to the list.

**Note:** I also recommend you to sort the packages in `requirements.txt` in alphabetical order, it saves time as the project evolves.
